playbooks/openstack-deploy_caasp.yml

```diff
@@ -43,3 +43,14 @@
       vars:
         skuba_cluster_basedir: "{{ socok8s_workspace }}"
         image_username: "{{ skuba_ci_terraform_openstack_image_username }}"
+
+- hosts: caasp-workers
+  vars_files:
+    - "{{ playbook_dir }}/../vars/common-vars.yml"
+  tasks:
+    - import_role:
+        name: dstat
+        tasks_from: main
+      when: lookup('env', 'SOCOK8S_DEPLOY_DSTAT') | default(False)
+      tags:
+        - yeah
```

Comment:
Yeah, why not!

Reply:
lmao, leftover of local testing, will remove!